modules/webxr/doc_classes/WebXRInterface.xml

```diff
@@ -105,6 +105,22 @@
 				Returns the display refresh rate for the current HMD. Not supported on all HMDs and browsers. It may not report an accurate value until after using [method set_display_refresh_rate].
 			</description>
 		</method>
+		<method name="get_hand_joint_radius" qualifiers="const">
+			<return type="float" />
+			<param index="0" name="input_source_id" type="int" />
+			<param index="1" name="hand_joint" type="int" enum="WebXRInterface.HandJoint" />
+			<description>
+				If hand tracking data is available, returns the radius of [param hand_joint] for the given [param input_source_id].
```

Comment:
May be worth linking the related methods together. That is, appending "see also get_hand_joint_transform and has_hand_tracking_data" at the end.

Reply:
This is now rebuilt on top of PR https://github.com/godotengine/godot/pull/88639 and so there aren't any doc changes specific to this PR (they all are all inherited from the other PR).